src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.serviceimpl.FruitTransactionValidatorImpl;
+import java.util.Optional;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
The class `ReturnOperationHandler` should not directly access the `Storage` class. Instead, consider using a service layer to interact with the storage to follow the SOLID principles, particularly the Single Responsibility Principle.